deploy/examples/create-external-cluster-resources.py

```diff
@@ -474,6 +474,24 @@ def gen_arg_parser(cls, args_to_parse=None):
             required=False,
             help="provides the name of the rgw-zonegroup",
         )
+        output_group.add_argument(
+            "--topology-pools",
+            default="",
+            required=False,
+            help="comma-separated list of topology-constrained pools",
+        )
+        output_group.add_argument(
```

Comment:
In the design should we add a common flag `--enable-topology` that will check for these three flags and errors if all three are not present?
Or this would be an unnecessary load, and if should just check all three are flags present and create the storage class JSON

Reply:
Seems like we don't need that other flag, we can just check if all three are present, if any of them are specified.

Reply:
Okay sure and also need to validate pool that it shouls actually exits in the ceph cluster

Reply:
We have a similar pattern for checking the existing of other pools in the external cluster, right?

Reply:
correct